data/esdt/esdt.pb.go

```diff
@@ -0,0 +1,1395 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: esdt.proto
+
+package esdt
+
+import (
+	bytes "bytes"
+	fmt "fmt"
+	github_com_ElrondNetwork_elrond_vm_common_data "github.com/ElrondNetwork/elrond-vm-common/data"
+	_ "github.com/gogo/protobuf/gogoproto"
+	proto "github.com/gogo/protobuf/proto"
+	io "io"
+	math "math"
+	math_big "math/big"
+	math_bits "math/bits"
+	reflect "reflect"
+	strings "strings"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
+
+// ESDigitalToken holds the data for a Elrond standard digital token transaction
+type ESDigitalToken struct {
+	Type          uint32        `protobuf:"varint,1,opt,name=Type,proto3" json:"Type"`
+	Value         *math_big.Int `protobuf:"bytes,2,opt,name=Value,proto3,casttypewith=math/big.Int;github.com/ElrondNetwork/elrond-vm-common/data.BigIntCaster" json:"Value"`
```

Comment:
we should test if this is breaking change or not (I think an import-db process will prove it is ok or not)